app_legacy/Helpers/render/layout.php

```diff
@@ -165,7 +165,7 @@ function RenderToolbar(): void
     echo "<li><a href='/forumposthistory.php'>Recent Forum Posts</a></li>";
     echo "<li class='divider'></li>";
     echo "<li><a href='/userList.php'>User List</a></li>";
-    echo "<li><a href='/globalRanking.php'>Global Ranking</a></li>";
+    echo "<li><a href='/globalRanking.php?s=5&t=2'>Global Ranking</a></li>";
```

Comment:
It would probably be more effective to change the default value for `t` in globalRanking.php to 2.

https://github.com/RetroAchievements/RAWeb/blob/664b00fed323b0a79c108231314dba7d9f4c408d/public/globalRanking.php#L12-L13

Reply:
Good point. I've addressed this and updated all these existing links on the site in https://github.com/RetroAchievements/RAWeb/pull/1598/commits/a281234cc0247249a8092904f6167b5c37c19011.